src/main/java/racingcar/domain/CarName.java

```diff
@@ -0,0 +1,50 @@
+package racingcar.domain;
+
+public class CarName {
+
+    private final String name;
+
+    private static final int MIN_LENGTH_OF_NAME = 1;
+    private static final int MAX_LENGTH_OF_NAME = 5;
+    private static final String EMPTY_CAR_NAME = "[ERROR] 자동차 이름이 입력되지 않았습니다.";
+    private static final String MORE_THAN_ONE_LETTER = "[ERROR] 1글자 미만의 자동차 이름은 허용되지 않습니다.";
+    private static final String LESS_THAN_FIVE_LETTER = "[ERROR] 5글자 초과의 자동차 이름은 허용되지 않습니다.";
+    private static final String NO_SPACE = "[ERROR] 공백을 포함하는 자동차 이름은 허용되지 않습니다.";
```

Comment:
일반적으로 상수가  필드보다 위에 위치합니다 :)

```suggestion
    private static final int MIN_LENGTH_OF_NAME = 1;
    private static final int MAX_LENGTH_OF_NAME = 5;
    private static final String EMPTY_CAR_NAME = "[ERROR] 자동차 이름이 입력되지 않았습니다.";
    private static final String MORE_THAN_ONE_LETTER = "[ERROR] 1글자 미만의 자동차 이름은 허용되지 않습니다.";
    private static final String LESS_THAN_FIVE_LETTER = "[ERROR] 5글자 초과의 자동차 이름은 허용되지 않습니다.";
    private static final String NO_SPACE = "[ERROR] 공백을 포함하는 자동차 이름은 허용되지 않습니다.";
    
    private final String name;
```

Reply:
상수 -> 클래스 변수 -> 인스턴스 변수 순서대로 정렬 하겠습니다.